docs/integrations/airflow.md

```diff
@@ -0,0 +1,128 @@
+# Airflow
+
+Airflow has a native OpenTelemetry integration for [traces] and [metrics].
+
+Airflow creates an exporter internally, which sends data to the configured backend.
+
+Unfortunately, it's not possible to send data to Logfire directly, yet... But we are working on it. :wink:
+
+??? question "Why it's not possible to send data directly?"
+    The Airflow implementation doesn't allow to override the headers used by the exporter.
+    See [apache/airflow#44346](https://github.com/apache/airflow/pull/44346) for more details.
+
+    We need the ability to set the authorization header to send data to **Logfire**.
+    See our [Alternative clients section](../guides/advanced/alternative-clients.md).
+
+## Setup
+
+To enable the OpenTelemetry integration in Airflow, you'll need to have an [OpenTelemetry Collector] running, and then
+configure Airflow to send data to it.
+
+### OpenTelemetry Collector
```

Comment:
This feels like it should be on the alternative clients page or something.

Reply:
Especially because there are other reasons to use a collector

Reply:
Hmmm... Alternative backends maybe?

Maybe just a `OpenTelemetry Collector` section? 🤔 And then we explain what you can do with the collector (what should I add? sampling?) and I can also explain it can be used to transition from the current backend to Logfire...

Reply:
Yes actually alternative backends makes more sense. Or a new page.

@adriangb can you propose anything to say about why a collector is a good idea?

I don't think we necessarily have to sell it. But it'd be nice for readers to have some idea why the page/section exists (without having to come from the airflow docs) and to feel like it's something they might want to look into if they have time but that it's not needed.

Reply:
I do think having an extensive section on the otel collector makes sense.

There's a lot of reasons you might want it including:
- centralizing configuration management including redaction, auth tokens for logfire or other upstream services, sending telemetry to multiple places, sampling to reduce costs, etc.
- scraping metrics from prometheus enabled services
- enriching with metadata like k8s host information

And more I don't want to go into in this comment. There's _a lot_ of talks, blog posts, etc. on the otel collector.

I'm happy to contribute to writing this section.

Reply:
I'd prefer to have this section as is right now, because at the time, it's needed.

My PR was merged on Airflow, so the collector will not be needed on the next airflow release.

But also... We can leave the collector setup that's written here till Adrian works on the new section.